platform/qt/src/qmapboxgl.cpp

```diff
@@ -273,8 +362,28 @@ void QMapboxGLSettings::setAccessToken(const QString &token)
 */
 
 /*!
+    \enum QMapboxGL::NorthOrientation
+
+    This enum sets the orientation of the north heading. It will directly affect bearing when
```

Comment:
Other documentation comments refer to "bearing" instead of "heading". The Qt SDK should standardize on one or the other. (For example, "bearing" is the standard term on Android, while "direction" and "heading" are the standard terms on Apple platforms.)